bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/deviceparameters/constants/ApplicationGroup.java

```diff
@@ -0,0 +1,101 @@
+package org.openhab.binding.digitalstrom.internal.lib.structure.devices.deviceparameters.constants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * digitalSTROM Application Groups.
+ * 
+ * <pre>
+	| Group ID | Name                  | Color   | Application                         |
+	| -------- | --------------------- | ------- | ----------------------------------- |
+	| 1        | Lights                | Yellow  | Room lights                         |
+	| 2        | Blinds                | Gray    | Blinds, curtains, shades, awnings   |
+	| 3        | Heating               | Blue    | Heating                             |
+	| 9        | Cooling               | Blue    | Cooling                             |
+	| 10       | Ventilation           | Blue    | Ventilation                         |
+	| 11       | Window                | Blue    | Windows                             |
+	| 12       | Recirculation         | Blue    | Ceiling fan, Fan coil units         |
+	| 64       | Apartment Ventilation | Blue    | Ventilation system                  |
+	| 48       | Temperature Control   | Blue    | Single room temperature control     |
+	| 4        | Audio                 | Cyan    | Playing music or radio              |
+	| 5        | Video                 | Magenta | TV, Video                           |
+	| 8        | Joker                 | Black   | Configurable                        |
+	| n/a      | Single Device         | White   | Various, individual per device      |
+	| n/a      | Security              | Red     | Security related functions, Alarms  |
+	| n/a      | Access                | Green   | Access related functions, door bell |
+ * </pre>
+ * 
+ * @author Rouven Schürch - Initial contribution
+ * @see <a href="https://developer.digitalstrom.org/Architecture/ds-basics.pdf">ds-basics.pdf</a> (Version 1.4/1.6),
+ *      chapter 3.2 (Group), Table 2.
+ *
+ */
+public enum ApplicationGroup {
+
+    LIGHTS((short) 1, Color.YELLOW),
+    BLINDS((short) 2, Color.GREY),
+    HEATING((short) 3, Color.BLUE),
+    COOLING((short) 9, Color.BLUE),
+    VENTILATION((short) 10, Color.BLUE),
+    WINDOW((short) 11, Color.BLUE),
+    RECIRCULATION((short) 12, Color.BLUE),
+    APARTMENT_VENTILATION((short) 64, Color.BLUE),
+    TEMPERATURE_CONTROL((short) 48, Color.BLUE),
+    AUDIO((short) 4, Color.CYAN),
+    VIDEO((short) 5, Color.MAGENTA),
+    JOKER((short) 8, Color.BLACK),
+    SINGLE_DEVICE((short) -1, Color.WHITE),
+    SECURITY((short) -2, Color.RED),
+    ACCESS((short) -3, Color.GREEN),
+    UNDEFINED(null, Color.UNDEFINED);
+
+    public enum Color {
+        YELLOW,
+        GREY,
+        BLUE,
+        CYAN,
+        MAGENTA,
+        BLACK,
+        WHITE,
+        RED,
+        GREEN,
+        UNDEFINED
+    }
+
+    private Short groupId;
```

Comment:
Primitive types should be used where possible.

Reply:
This is by design, so the
```java
UNDEFINED(null, Color.UNDEFINED);
```
Enum can be constructed.